lib/rules/amd-function-arity.js

```diff
@@ -34,10 +34,6 @@ module.exports = function (context) {
         return node.arguments.filter(isFunctionExpression).length;
```

Comment:
Wouldn't it be enough to replace it by `util.isAmdRequire()`? But it's imho a different refactoring.